### DISCUSSION TITLE
Can I set my execution policy back to what it was after installing Scoop?

### DISCUSSION THREAD

**Discussion by LikeLakers2** [ScoopInstaller/Scoop#5182] [Help and Questions] [answered]
I want to install Scoop, but I'm concerned about changing the execution policy permanently.

Is it possible to set the execution policy, install Scoop, and then set it back? Or does the execution policy need to stay as Unrestricted/RemoteSigned/ByPass for me to even use Scoop?

**Comment by rashil2000** [comment, marked as answer]
I'm not entirely sure, but it's very simple to try it: Just revert it to what it was before installing Scoop. And now try running any Scoop commands, like 'install' or 'update'.

**Reply by LikeLakers2** [reply]
Just tried it, seems scoop doesn't work with it reverted.

However, I realized an easy solution to what I wanted to do (which is prevent scripts other than scoop from running). Whenever I want to run scoop, I just set the Execution Policy to RemoteSigned, do whatever I need to with scoop, and then set it back.

**Comment by Vzz1c** [comment]
Get-ExecutionPolicy -List

Set-ExecutionPolicy -ExecutionPolicy Undefined -Scope xxxxx
